packages/amplify-graphql-api-construct-tests/src/__tests__/backends/sql-models/app.ts

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import { App, Stack, Duration } from 'aws-cdk-lib';
+// @ts-ignore
+import {
+  AmplifyGraphqlApi,
+  AmplifyGraphqlDefinition,
+  SqlModelDataSourceBindingDbConnectionConfig,
+} from '@aws-amplify/graphql-api-construct';
+
+interface DBDetails {
```

Comment:
This is a copy/paste of the same interface from the test code. I played with a couple of ways to factor it into a standalone file, but couldn't figure out a way I liked to copy it into the app.ts workspace to make it available for both test-time and compile-time use. Open to suggestions.